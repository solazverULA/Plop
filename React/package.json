{
  "name": "nud-react",
  "version": "1.0.0",
  "private": true,
  "homepage": "https://timcreative.github.io/now-ui-dashboard-react",
  "dependencies": {
    "animate.css": "^3.6.1",
    "babel-preset-env": "^1.7.0",
    "babel-preset-react": "^6.24.1",
    "bootstrap": "4.0.0-beta",
    "chart.js": "^2.7.2",
    "history": "4.7.2",
    "jquery": "3.3.1",
    "node-sass": "^4.9.0",
    "node-sass-chokidar": "^1.3.0",
    "npm-run-all": "4.1.1",
    "perfect-scrollbar": "1.3.0",
    "react": "16.1.0",
    "react-animated-css": "^1.0.4",
    "react-chartjs-2": "2.7.0",
    "react-dom": "16.1.0",
    "react-google-maps": "9.2.2",
    "react-notification-alert": "0.0.7",
    "react-router-dom": "4.2.2",
    "react-scripts": "^1.1.4",
    "reactstrap": "5.0.0-alpha.4"
  },
  "devDependencies": {
    "babel-jest": "^22.0.4",
    "jest": "^22.0.4",
    "react-test-renderer": "^16.0.0"
  },
  "scripts": {
    "build-css": "node-sass-chokidar --include-path ./src --include-path ./node_modules src/ -o src/",
    "watch-css": "npm run build-css && node-sass-chokidar --include-path ./src --include-path ./node_modules src/ -o src/ --watch --recursive",
    "start-js": "react-scripts start",
    "start": "npm-run-all -p watch-css start-js",
    "build": "npm run build-css && react-scripts build",
    "test": "jest",
    "jest-update-snapshot": "jest --updateSnapshot",
    "eject": "react-scripts eject"
  }
}
