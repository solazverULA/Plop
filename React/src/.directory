[Dolphin]
Timestamp=2018,6,27,3,57,52
Version=3
