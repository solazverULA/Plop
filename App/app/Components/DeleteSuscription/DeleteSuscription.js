import React, { Component } from 'react';
import {
  Platform,
  StyleSheet,
  Text,
  View,
  AsyncStorage
} from 'react-native';

import SeeProfile from "../SeeProfile/SeeProfile"
import Modal from 'react-native-modal';
import { Button } from 'react-native-elements'
export default class DeleteSuscription extends Component<{}> {
  static navigationOptions = {
    drawerLabel: 'Tus subscripciones',
    
    
  };

  constructor(props) {
    super(props);
    
    this.state = {indexOpen:0,ModalOpen:false};
  }

  _toggleModal(index){
    this.setState({...this.state,ModalOpen:!this.state.ModalOpen,indexOpen:index})
  }
  _Loguot(){
    AsyncStorage.removeItem("ListenerData")
      .then(()=>AsyncStorage.removeItem("ListenerNotifications"))
        .then(()=>this.props.navigation.navigate('Login'))
  }

  render() {
    return (
      <View style={styles.container}>
        <Text>Aqui puedes eliminar la suscripcion si lo deseas</Text>      

        <Button
          raised
          icon={{name: 'cached'}}
          title='eliminar' 
          onPress={()=>this._toggleModal(0)}
          />
        <Modal isVisible={this.state.ModalOpen } onBackdropPress={()=>this._toggleModal(0)}>
          <Text>Estas segura que lo quieres hacer?</Text>  
           <Button
          raised
          icon={{name: 'cached'}}
          title='eliminar'
          onPress={this._Loguot.bind(this)}
           />
        </Modal>
         
          
        
        
      </View>
    );
  }
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
   
    backgroundColor: '#F5FCFF',
  },
  welcome: {
    fontSize: 20,
    textAlign: 'center',
    margin: 10,
  },
  instructions: {
    textAlign: 'center',
    color: '#333333',
    marginBottom: 5,
  },

 

  Text:{
    color:"white"
  }
});
