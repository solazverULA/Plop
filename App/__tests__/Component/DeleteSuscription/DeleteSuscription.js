import 'react-native';
import React from 'react';
import DeleteSuscription from '../../../app/Components/DeleteSuscription/DeleteSuscription';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const loading = renderer.create(
    <DeleteSuscription />
  ).toJSON();
  
  expect(loading).toMatchSnapshot();
});
