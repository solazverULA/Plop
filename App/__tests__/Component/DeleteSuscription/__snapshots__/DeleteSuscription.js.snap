// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#F5FCFF",
      "flex": 1,
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
  >
    Aqui puedes eliminar la suscripcion si lo deseas
  </Text>
  <View
    style={
      Array [
        Object {
          "marginLeft": 15,
          "marginRight": 15,
        },
        Object {
          "shadowColor": "rgba(0,0,0, .4)",
          "shadowOffset": Object {
            "height": 1,
            "width": 1,
          },
          "shadowOpacity": 1,
          "shadowRadius": 1,
        },
        undefined,
        undefined,
      ]
    }
  >
    <View
      accessibilityComponentType={undefined}
      accessibilityLabel={undefined}
      accessibilityTraits={undefined}
      accessible={true}
      hasTVPreferredFocus={undefined}
      hitSlop={undefined}
      isTVSelectable={true}
      nativeID={undefined}
      onLayout={undefined}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Array [
          Object {
            "backgroundColor": "transparent",
          },
          undefined,
        ]
      }
      testID={undefined}
      tvParallaxProperties={undefined}
    >
      <View
        pointerEvents="box-only"
        style={
          Array [
            Object {
              "alignItems": "center",
              "backgroundColor": "#9E9E9E",
              "flexDirection": "row",
              "justifyContent": "center",
              "padding": 19,
            },
            undefined,
            undefined,
            undefined,
            undefined,
            undefined,
            undefined,
            undefined,
            Object {
              "padding": 12,
            },
            undefined,
            undefined,
            undefined,
            undefined,
            undefined,
            undefined,
          ]
        }
      >
        <Text
          accessible={true}
          allowFontScaling={false}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "white",
                "fontSize": 18,
              },
              Array [
                Object {
                  "marginRight": 10,
                },
                undefined,
              ],
              Object {
                "fontFamily": "Material Icons",
                "fontStyle": "normal",
                "fontWeight": "normal",
              },
            ]
          }
        >
          
        </Text>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {},
              undefined,
              undefined,
              undefined,
              undefined,
              undefined,
              undefined,
              undefined,
              undefined,
              undefined,
              Array [
                Object {
                  "color": "white",
                  "fontSize": 20,
                },
                undefined,
                Object {
                  "fontSize": 17.5,
                },
                undefined,
                undefined,
                undefined,
                undefined,
              ],
            ]
          }
        >
          eliminar
        </Text>
      </View>
    </View>
  </View>
  <Modal
    animationType="none"
    hardwareAccelerated={false}
    onModalHide={[Function]}
    onRequestClose={[Function]}
    transparent={true}
    visible={false}
  >
    <View
      accessibilityComponentType={undefined}
      accessibilityLabel={undefined}
      accessibilityTraits={undefined}
      accessible={true}
      collapsable={undefined}
      hitSlop={undefined}
      nativeID={undefined}
      onLayout={undefined}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "backgroundColor": "black",
          "bottom": 0,
          "height": 1334,
          "left": 0,
          "opacity": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
          "width": 750,
        }
      }
      testID={undefined}
    />
    <View
      collapsable={undefined}
      onModalHide={[Function]}
      pointerEvents="box-none"
      style={
        Object {
          "flex": 1,
          "justifyContent": "center",
          "margin": 37.5,
          "transform": Array [
            Object {
              "translateY": 0,
            },
          ],
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
      >
        Estas segura que lo quieres hacer?
      </Text>
      <View
        style={
          Array [
            Object {
              "marginLeft": 15,
              "marginRight": 15,
            },
            Object {
              "shadowColor": "rgba(0,0,0, .4)",
              "shadowOffset": Object {
                "height": 1,
                "width": 1,
              },
              "shadowOpacity": 1,
              "shadowRadius": 1,
            },
            undefined,
            undefined,
          ]
        }
      >
        <View
          accessibilityComponentType={undefined}
          accessibilityLabel={undefined}
          accessibilityTraits={undefined}
          accessible={true}
          hasTVPreferredFocus={undefined}
          hitSlop={undefined}
          isTVSelectable={true}
          nativeID={undefined}
          onLayout={undefined}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Array [
              Object {
                "backgroundColor": "transparent",
              },
              undefined,
            ]
          }
          testID={undefined}
          tvParallaxProperties={undefined}
        >
          <View
            pointerEvents="box-only"
            style={
              Array [
                Object {
                  "alignItems": "center",
                  "backgroundColor": "#9E9E9E",
                  "flexDirection": "row",
                  "justifyContent": "center",
                  "padding": 19,
                },
                undefined,
                undefined,
                undefined,
                undefined,
                undefined,
                undefined,
                undefined,
                Object {
                  "padding": 12,
                },
                undefined,
                undefined,
                undefined,
                undefined,
                undefined,
                undefined,
              ]
            }
          >
            <Text
              accessible={true}
              allowFontScaling={false}
              ellipsizeMode="tail"
              style={
                Array [
                  Object {
                    "color": "white",
                    "fontSize": 18,
                  },
                  Array [
                    Object {
                      "marginRight": 10,
                    },
                    undefined,
                  ],
                  Object {
                    "fontFamily": "Material Icons",
                    "fontStyle": "normal",
                    "fontWeight": "normal",
                  },
                ]
              }
            >
              
            </Text>
            <Text
              accessible={true}
              allowFontScaling={true}
              ellipsizeMode="tail"
              style={
                Array [
                  Object {},
                  undefined,
                  undefined,
                  undefined,
                  undefined,
                  undefined,
                  undefined,
                  undefined,
                  undefined,
                  undefined,
                  Array [
                    Object {
                      "color": "white",
                      "fontSize": 20,
                    },
                    undefined,
                    Object {
                      "fontSize": 17.5,
                    },
                    undefined,
                    undefined,
                    undefined,
                    undefined,
                  ],
                ]
              }
            >
              eliminar
            </Text>
          </View>
        </View>
      </View>
    </View>
  </Modal>
</View>
`;
